src/main/java/gift/controller/MemberController.java

```diff
@@ -0,0 +1,51 @@
+package gift.controller;
+
+import gift.dto.AuthResponseDto;
+import gift.dto.MemberRequestDto;
+import gift.service.MemberService;
+import org.apache.coyote.BadRequestException;
+import org.apache.tomcat.websocket.AuthenticationException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.validation.annotation.Validated;
+import org.springframework.web.bind.annotation.*;
+
+@RequestMapping("/api/members")
+@RestController
+public class MemberController {
+
+    private final MemberService memberService;
+    private static final Logger log = LoggerFactory.getLogger(MemberController.class);
```

Comment:
```suggestion
    private static final Logger log = LoggerFactory.getLogger(MemberController.class);
    
    private final MemberService memberService;

```

자바 관습적으로 상수는 맨 위에 적기는 합니다😃

Reply:
넵 반영하겠습니다!